test/test_cli.py

```diff
@@ -136,3 +139,66 @@ def test_should_find_root_package_when_root_used_project_name_used(self):
 
         # then
         self.assertEqual(test_module_src, resolve(res))
+
+    def test_should_run_workflow(self):
+        # given
+        root_package = find_root_package(None, "test.test_module")
+
+        # when
+        cli_run(root_package, workflow_id="ID_3")
+
+        # then
+        self.assert_started_jobs(['J_ID_3', 'J_ID_4'])
+
+        # when
+        cli_run(root_package, workflow_id="ID_4")
+
+        # then
+        self.assert_started_jobs(['J_ID_3', 'J_ID_4', 'J_ID_5'])
+
+    def test_should_run_workflow_multiple_times(self):
```

Comment:
Do we need a separate test for running multiple workflows?

Reply:
No, but I separated it to have more smaller tests instead of one big